modules/phase_field/include/ics/MultiSmoothSuperellipsoidIC.h

```diff
@@ -41,8 +41,9 @@ class MultiSmoothSuperellipsoidIC : public SmoothSuperellipsoidBaseIC
   Real _gk;
 
   MooseEnum _semiaxis_variation_type;
-  MooseEnum _prevent_overlap;
-  MooseEnum _check_extremes;
+  bool _prevent_overlap;
+  bool _check_extremes;
+  bool _vary_axes_independently;
```

Comment:
make cll those bools `const` please. They are not supposed to change after initialization.